src/swap/components/SwapProvider.tsx

```diff
@@ -36,9 +36,11 @@ export function useSwapContext() {
 export function SwapProvider({
   address,
   children,
+  useAggregator,
```

Comment:
the provider too should have `experimental`

Reply:
we don't want to leak bad experience across too many components, and we want to keep `experimental` nice and neat.